docs/esi-leap-requirements.md

```diff
@@ -0,0 +1,47 @@
+# ESI Leap Requirements
+
+## Overview
+ESI Leap is a leasing service for bare metal. Ironic nodes owners can offer up their nodes to other users, while leasees can view offers and create a contract for available nodes.
+
+## Definitions
+
+### Offer
+Ironic owners can bring up their nodes to ESI Leap with an offer. An offer consists of nodes configurations, leasing period, and the price, etc. An offer can be created, destroyed, saved, retrieved, and expired by ESI Leap. Here is an example offer schema:
```

Comment:
The offer doesn't really specify a node configuration; it specifies a node (through the resource_uuid). I think we should also eliminate references to 'price'.